src/Spillgebees.Blazor.Map.Assets/src/styles.scss

```
@@ -269,19 +271,24 @@
 // pseudo-fullscreen fallback for browsers without the Fullscreen API (e.g. iPhone Safari);
 // maplibre resizes the canvas via its container ResizeObserver once the element fills the viewport
 .sgb-map-container.sgb-map-pseudo-fullscreen {
+  // biome-ignore lint/complexity/noImportantStyles: fallback must win over host layout and MapLibre container styles
   position: fixed !important;
+  // biome-ignore lint/complexity/noImportantStyles: fallback must pin the map to the viewport regardless of host layout
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)